docs/internals/contributing/triaging-tickets.txt

```diff
@@ -414,8 +414,7 @@ Bisecting a regression
 
 .. highlight:: console
 
-A regression is a bug that's present in some newer version of Django but not in
-an older one. An extremely helpful piece of information is the commit that
+Regression is a bug that's present in some newer versions of Django but not in older ones. An extremely helpful piece of information is the commit that
```

Comment:
Why this edit? Please revert.

Reply:
I think it makes sense. There are multiple older versions :D

Reply:
Oh, I didn't even notice that part. I was looking at "Regression" instead of "A regression". "Regression is a bug" is not quite right.

Reply:
Changing "Regression" to "A regression" :p